src/test/java/seedu/address/testutil/TypicalStudents.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.testutil;
+
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_FRIEND;
+import static seedu.address.logic.commands.StudentTestUtil.VALID_ADDRESS_AARON;
+import static seedu.address.logic.commands.StudentTestUtil.VALID_ADDRESS_BILL;
+import static seedu.address.logic.commands.StudentTestUtil.VALID_EMAIL_AARON;
+import static seedu.address.logic.commands.StudentTestUtil.VALID_EMAIL_BILL;
+import static seedu.address.logic.commands.StudentTestUtil.VALID_NAME_AARON;
+import static seedu.address.logic.commands.StudentTestUtil.VALID_NAME_BILL;
+import static seedu.address.logic.commands.StudentTestUtil.VALID_PHONE_AARON;
+import static seedu.address.logic.commands.StudentTestUtil.VALID_PHONE_BILL;
+import static seedu.address.logic.commands.StudentTestUtil.VALID_STUDENT_ID_AARON;
+import static seedu.address.logic.commands.StudentTestUtil.VALID_STUDENT_ID_BILL;
+import static seedu.address.logic.commands.StudentTestUtil.VALID_TAG_STUDENT;
+import static seedu.address.logic.commands.TutorialTestUtil.VALID_TUTORIAL_NAME_TG1;
+import static seedu.address.logic.commands.TutorialTestUtil.VALID_TUTORIAL_NAME_TG2;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.person.Student;
+
+public class TypicalStudents {
+
+    public static final Student ALEX = new StudentBuilder().withName("Alex Lee")
+            .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alex@example.com")
+            .withPhone("91234567").withTags("student").withStudentId("e0543216")
+            .withTutorialName("TG01").build();
+    public static final Student BERNARD = new StudentBuilder().withName("Bernard Toh")
+            .withAddress("123, Wood East Ave 6, #08-123").withEmail("bernardt@gmail.com")
+            .withPhone("81234567").withTags("student").withStudentId("e0523416")
+            .withTutorialName("TG01").build();
+    public static final Student CHARLIE = new StudentBuilder().withName("Charlie Johnson")
+            .withAddress("321, Jurong North Ave 9, #10-123").withEmail("cjohnson@hotmail.com")
+            .withPhone("83214567").withTags("student").withStudentId("e0523614")
+            .withTutorialName("TG01").build();
+    public static final Student DENSON = new StudentBuilder().withName("Denson Loh")
+            .withAddress("321, Ang Mo Kio West Ave 9, #10-101").withEmail("dsnlh@hotmail.com")
+            .withPhone("83214567").withTags("student").withStudentId("e0563214")
+            .withTutorialName("TG01").build();
+
+    // Manually added - Student's details found in {@code StudentTestUtil}
```

Comment:
im not too clear why this is needed, but if you think it should remain then ok

Reply:
These are the manually added objects whose fields are valid and will be available in their respective class' Test Util to be used for testing. In future, these Test Util files will also contain invalid fields for testing as well.